docker/kubernetes-tentacle/Dockerfile

```diff
@@ -53,7 +53,8 @@ ENV OCTOPUS__K8STENTACLE__DISABLEAUTOPODCLEANUP="False"
 ENV TentacleHome=""
 ENV TentacleApplications=""
 
-CMD /scripts/configure-tentacle.sh && /scripts/run-tentacle.sh
+# Change the entrypoint to be tini
```

Comment:
Maybe update this comment to a "why" instead of a "what"?

Reply:
yep, good call!

Reply:
Sorry, I added another comment for the other exec part too 😅

Reply:
done!